### PR TITLE
Configure ruby kafka logger

### PR DESCRIPTION
**Pull request by tulios**
This pull request fixes issue #23

### CODE REVIEW

File: spec/lib/phobos_spec.rb [hunk 1]
```diff
@@ -11,16 +11,34 @@
...
     it 'returns a new kafka client already configured' do
-      Phobos.configure(phobos_config_path)
+      Phobos.config.logger.ruby_kafka = nil
+      Phobos.configure_logger
 
       expect(Kafka)
         .to receive(:new)
-        .with(hash_including(Phobos.config.kafka.to_hash))
+        .with(hash_including(Phobos.config.kafka.to_hash.merge(logger: nil)))
         .and_return(:kafka_client)
```
Comment (klippx) [inline spec/lib/phobos_spec.rb]: maybe it doesnt need to return `:kafka_client`, which is a bit of a no op and also make it look like the test below

File: spec/lib/phobos_spec.rb [hunk 2]
```diff
@@ -11,16 +11,34 @@
...
+    describe 'when "logger.ruby_kafka" is configured' do
+      before do
+        Phobos.config.logger.ruby_kafka = Phobos::DeepStruct.new(level: 'info')
+        Phobos.configure_logger
+      end
+
+      it 'configures "logger"' do
+        expect(Kafka)
+          .to receive(:new)
+          .with(hash_including(logger: instance_of(Logging::Logger)))
```
Comment (klippx) [inline spec/lib/phobos_spec.rb]: Does it work if you write it like the test above with a syntax as `.with(hash_including(Phobos.config.kafka.to_hash.merge(logger: instance_of(Logging::Logger))))`  ?

File: spec/lib/phobos_spec.rb (hunk 1 above)
Comment (tulios) [inline spec/lib/phobos_spec.rb]: It's just to ensure that the method returns the kafka client

Comment (klippx) [inline spec/lib/phobos_spec.rb]: OK then, cool!

**Pull request merged**